src/telliot_feeds/reporters/rng_interval.py

```diff
@@ -313,8 +313,10 @@ async def report(self) -> None:
         logger.info(f"RNG reporting interval: {INTERVAL} seconds")
         while True:
             online = await is_online()
-            if not online:
+            if online:
+                _, _ = await self.report_once()
+                logger.info(f"Sleeping for {self.wait_period} seconds")
```

Comment:
move the sleeping msg out of the condition and put above where it sleeps. It's going to sleep whether it's online or not